### PR TITLE
docs: update README

### LINKED ISSUE #3145: [Docs]: update repo readme file
### Summary

Some links in the repo's readme.md might be out of date or out of place.

Update the readme by removing the outdated sections and add sections as necessary.

### PR DESCRIPTION
**Pull request by mpoke**
## Description

Closes: #3145

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct `docs:` prefix in the PR title
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,117 +11,47 @@ developed.](https://img.shields.io/badge/repo%20status-Active-green.svg)](https:
...
+For an overview of the Interchain Stack (the Cosmos SDK, IBC, etc.), as well as examples and exercises to help developers get a quick start, please visit [tutorials.cosmos.network](https://tutorials.cosmos.network).
 
-## 💬 — Talk to us
+### Additional resources
 
-___
+**For node operators:** If you're interested in running a node on the current Cosmos Hub, check out the docs to [Join the Cosmos Hub Mainnet](https://hub.cosmos.network/main/hub-tutorials/join-mainnet).
 
-We have active, helpful communities on Twitter, Discord, and Telegram.
+**For validators:** If you want to participate and help secure the Cosmos Hub and its [consumer chains](https://hub.cosmos.network/main/interchain-security), check out becoming a validator. Information on what a validator is and how to participate as one can be found at the [Validator FAQ](https://hub.cosmos.network/main/validators/validator-faq). If you're running a validator node on the Cosmos Hub, reach out to a Janitor on the [Cosmos Developers Discord](https://discord.gg/interchain) to join the `#cosmos-hub-validators-verified` channel.
```
Comment (mpoke) [inline README.md]: @MSalopek @Btruax is this info accurate?

Comment (insumity) [inline README.md]: nit: s/check out becoming a validator/consider becoming a validator

File: README.md [hunk 2]
```diff
@@ -11,117 +11,47 @@ developed.](https://img.shields.io/badge/repo%20status-Active-green.svg)](https:
...
 
-|    |    |
-| -- | -- |
-| Cosmos Developers Discord | <a href="https://discord.gg/cosmosnetwork"><img src="https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white" alt="Discord" height="30"/></a> |
-| Cosmos Twitter | <a href="https://twitter.com/cosmos"><img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" alt="Tweet" height="30"/></a> |
-| Cosmos Gov Twitter | <a href="https://twitter.com/CosmosGov"><img src="https://img.shields.io/badge/Twitter-FEFEFE?style=for-the-badge&logo=twitter&logoColor=blue" alt="Tweet" height="30"/></a> |
-| Cosmos Telegram | <a href="https://t.me/cosmosproject"><img src="https://img.shields.io/badge/Telegram-2CA5E0?style=for-the-badge&logo=telegram&logoColor=white" alt="Telegram" height="30"/></a> |
+**For delegators:** If you still want to participate on the Cosmos Hub, check out becoming a delegator. Information on what a delegator is and how to participate as one can be found at the [Delegator FAQ](https://hub.cosmos.network/main/delegators/delegator-faq).
```
Comment (insumity) [inline README.md]: Not sure about the "still"? Maybe you could say something similar to: "If you want to secure the Cosmos Hub without running a validator, you can consider becoming a delegator."

File: README.md [hunk 3]
```diff
@@ -11,117 +11,47 @@ developed.](https://img.shields.io/badge/repo%20status-Active-green.svg)](https:
 [![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=cosmos_gaia&metric=ncloc)](https://sonarcloud.io/summary/new_code?id=cosmos_gaia)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=cosmos_gaia&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=cosmos_gaia)
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=cosmos_gaia&metric=coverage)](https://sonarcloud.io/summary/new_code?id=cosmos_gaia)
-[![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/cosmosnetwork)
-[![Twitter](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/cosmoshub)
+[![Twitter](https://badgen.net/badge/icon/Twitter?icon=twitter&label)](https://x.com/cosmoshub)
```
Comment (insumity) [inline README.md:14]: If we're changing the URL to X 😢 , then maybe we can change the icon as well.

File: README.md [hunk 4]
```diff
@@ -11,117 +11,47 @@ developed.](https://img.shields.io/badge/repo%20status-Active-green.svg)](https:
...
+## Contributing
 
-<br/>
+Check out [CONTRIBUTING.md](CONTRIBUTING.md) for our guidelines & policies for how we develop the Cosmos Hub. Thank you to all those who have contributed!
 
-<p align="center"><a href="https://github.com/nastyox/Rando.js#nastyox"><img src="http://randojs.com/images/barsSmallTransparentBackground.gif" alt="Animated footer bars" width="100%"/></a></p>
+## Talk to us
 
-<br/>
+Follow the Cosmos Hub team's activities on the [Cosmos Hub Twitter](https://twitter.com/cosmoshub) account.
```
Comment (insumity) [inline README.md]: Maybe use `x.com` here as well, similar to what you have above.

File: docs/docs/delegators/delegator-guide-cli.md [hunk 5]
```diff
@@ -7,45 +7,32 @@ This document contains all the necessary information for delegators to interact
 
 It also contains instructions on how to manage accounts, restore accounts from the fundraiser and use a ledger nano device.
 
-:::warning
-**Very Important**: Please assure that you follow the steps described hereinafter
```
Comment (insumity) [inline docs/docs/delegators/delegator-guide-cli.md:11]: Should we keep some type of a disclaimer?

**Review by insumity — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (mpoke) [reply README.md:14]: I cannot find an icon

File: docs/docs/delegators/delegator-guide-cli.md (hunk 5 above)
Comment (mpoke) [reply docs/docs/delegators/delegator-guide-cli.md:11]: @MSalopek asked me to remove it.

Comment (MSalopek) [reply docs/docs/delegators/delegator-guide-cli.md:11]: Not to remove it, bump it down to a `:::tip` or `:::info` so it's not red

File: README.md (hunk 1 above)
Comment (MSalopek) [reply README.md]: Don't know what's a Janitor, but I suppose so.

File: docs/docs/delegators/delegator-guide-cli.md [hunk 6]
```diff
@@ -7,45 +7,32 @@ This document contains all the necessary information for delegators to interact
 
 It also contains instructions on how to manage accounts, restore accounts from the fundraiser and use a ledger nano device.
 
-:::warning
```
Comment (MSalopek) [inline docs/docs/delegators/delegator-guide-cli.md:10]: Leave the banner, but please change severity to `:::tip`

**Review by MSalopek — APPROVED** [review]
LGTM.

Nits can be fixed later, same for broken links.

**Pull request merged**